### ISSUE TITLE
Object Types as Query Arg is not parsed correctly

### ISSUE THREAD

**Issue by tibawatanabe** [Urigo/SOFA#28]
I'm trying to pass an objetc as a query parameters:

```gql
input PageInfoType {
  offset: Int!
  limit: Int!
}

type Query {
  users(pageInfo: PageArgsType!): [UsersType]!
}
```

I'm unable to consume the endpoint because Sofa is not parsing the query params correctly.

When executing, the query is receiving `{ pageInfo: '{ "limit": 5 }' }` instead of `{ pageInfo: { limit: 5 } }` causing the following error: 

```
Variable "$pageInfo" got invalid value "{ \"limit\": 5 }";
Expected type PageArgsType to be an object
```

I'm not sure if this is related but the swagger points the following error as well:

```
Resolver error at paths./api/users.get.parameters.0.schema.$ref
Could not resolve reference: #/components/schemas/PageArgsType
```

And indeed, the input type is not listed in the generated json.

**Comment by kamilkisiela** [comment]
Fix released in `v0.3.0`

**Comment by hajnalben** [comment]
Maybe the queries with input type parameter (which is technically an object) could be generated as POST methods? It would make more sense than passing an object in query.
